spec/lib/cli_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe Rambo::CLI do
+RSpec.describe Rambo::CLI do
```

Comment:
Oh! So does this fix the behavior where it actually changes the spec helper file?